sys/log/full/selftest/util/src/log_test_util.c

```diff
@@ -130,18 +130,20 @@ ltu_walk_verify(struct log *log, struct log_offset *log_offset,
     struct os_mbuf *om;
     char data[128];
     int dlen;
+    uint16_t ext_len;
```

Comment:
Bit of a nitpick, but I feel like this variable is misnamed.  It contains the full header length, not just the extra length.  I would expect it to be called `hdr_len`.